src/components/WwwFrame/TheBasketBar.vue

```diff
@@ -41,6 +41,7 @@ export default {
 	background-color: $kiva-accent-blue;
 	font-size: rem-calc(20);
 	font-weight: normal;
+	z-index: 1000;
```

Comment:
Recent changes to the category rows carousel introduced z-indexes that placed them above the basket bar. This fixes that.